README.rst

```diff
@@ -1,104 +1,136 @@
+
+Usage
+=====
+
+Once your instance is running, you can access the Django admin site and locate the ``EDUNEXT OPENEDX MULTITENANCY`` models.
+
+- **Microsites:** Store the microsite configuration.
+- **Routes:** Configure the URL for a tenant.
+- **Tenant configs:** Store the configuration for each tenant.
+- **Tenant organizations:** Link each organization with one or multiple tenants.
+
+Add ``EDNX_USE_SIGNAL = True`` in each microsite/tenant that wants to use the plugin. 
 
 Compatibility Notes
 --------------------
 
-+------------------+-----------------------+
-| Open edX Release | Version               |
-+==================+=======================+
-| Ironwood         | < 3.0                 |
-+------------------+-----------------------+
-| Juniper          | >= 3.0 < 4.0          |
-+------------------+-----------------------+
-| Koa              | >= 4.0 <= 5.1.3       |
-+------------------+-----------------------+
-| Lilac            | >= 4.0 < 6.2          |
-+------------------+-----------------------+
-| Maple            | >= 6.0                |
-+------------------+-----------------------+
-| Nutmeg           | >= 6.2                |
-+------------------+-----------------------+
-| Olive            | >= 8.0                |
-+------------------+-----------------------+
-| Palm             | >= v10.0.0            |
-+------------------+-----------------------+
-| Quince           | >= v11.0.0            |
-+------------------+-----------------------+
-
-**NOTE**: Since 6.2 version, eox-tenant does not support Django 2.2
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
++------------------+----------------+
+| Open edX Release | Version        |
++==================+================+
+| Ironwood         | < 3.0          |
++------------------+----------------+
+| Juniper          | >= 3.0 < 4.0   |
++------------------+----------------+
+| Koa              | >= 4.0 <= 5.1.3|
++------------------+----------------+
+| Lilac            | >= 4.0 < 6.2   |
++------------------+----------------+
+| Maple            | >= 6.0 < 11.6  |
++------------------+----------------+
+| Nutmeg           | >= 6.2         |
++------------------+----------------+
+| Olive            | >= 8.0         |
++------------------+----------------+
+| Palm             | >= v10.0.0     |
++------------------+----------------+
+| Quince           | >= v11.0.0     |
++------------------+----------------+
+
+⚠️ Since the 6.2 version, eox-tenant does not support Django 2.2
+
+The plugin is configured for the latest release (Quince). The following changes in the plugin settings should be applied to be used for previous releases.
+
+If the release you are looking for is not listed, the actual configuration is sufficient or it is incompatible with the current eox-tenant version.
```

Comment:
This phrase can be improved for better understanding
```suggestion
If the release you are looking for is not listed, then either the current configuration is sufficient, or the release is incompatible with the current eox-tenant version.
```

Reply:
@dcoa even with these improvements, I'm honestly still having a hard time trying to understand what you explained in the comments and the purpose of this phrase

We can easily check if a release is compatible or not thanks to the [table](https://github.com/eduNEXT/eox-tenant/pull/210/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR48), right? should we specify this here? something like:  _or the release is incompatible with the current eox-tenant version (please refer to the table for compatibillity)._

Reply:
I agree, since the table defines the range of versions supported by a release,